backend/capellacollab/tools/models.py

```diff
@@ -1,4 +1,5 @@
 # SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
```

Comment:
There is one license line too many.
Also, the `ToolDockerimage` currently uses the `docker_image_template` as readonly (in `from_orm`, that is used under the hood everytime a FastAPI route has a response_model set.), I don’t think that’s the desired behaviour.